v2/samples/insights/v1api/v1api20240101preview_scheduledqueryrule.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: insights.azure.com/v1api20240101preview
+kind: ScheduledQueryRule
+metadata:
+  name: aso-sample-rule
+  namespace: default
+spec:
+  criteria:
+    allOf:
+    - failingPeriods:
+        minFailingPeriodsToAlert: 1
+        numberOfEvaluationPeriods: 1
+      operator: LessThan
+      query: "requests | summarize CountByCountry=count() by client_CountryOrRegion"
+      threshold: 10
+      timeAggregation: Count
+  evaluationFrequency: PT10M
+  identity:
+    type: UserAssigned
```

Comment:
Would this require a ResourceRef to an Identity?

Reply:
updated the sample to include a reference to an UserAssigned managed identity which is managed by ASO